<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsoleWindow</class>
 <widget class="QMainWindow" name="ConsoleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConsoleWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
border-radius:5px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{ 
	border-image: url(:/res/usr/menu_B.jpg);
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,5">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>268</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>华光粗圆_CNKI</family>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgba(255, 255, 255,0.9);</string>
        </property>
        <property name="text">
         <string>食记智能厨房KDS</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>华光中圆_CNKI</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>版本号：1.0.0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="userlabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>29</width>
                <height>29</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>微软雅黑 Light</family>
                <pointsize>15</pointsize>
                <weight>75</weight>
                <bold>true</bold>
                <underline>true</underline>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgba(255, 255, 255,0);
border-image: url(:/res/usr/usr.png);
color: rgb(255, 255, 255);
border-radius:30px;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>华光粗圆_CNKI</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgba(255, 255, 255,0.9);</string>
                </property>
                <property name="text">
                 <string>   您好</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="userEdit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>华光粗圆_CNKI</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-style:none;
color: rgba(255, 255, 255,0.9);
background-color: rgba(255, 255, 255,0);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="placeholderText">
                 <string>管理员</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>华光粗圆_CNKI</family>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgba(255, 255, 255,0.9);</string>
            </property>
            <property name="text">
             <string>本机IP：192.168.5.149</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius:20px;</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" rowspan="5">
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(57, 185, 243,0.9);</string>
         </property>
         <property name="text">
          <string>历史单据</string>
         </property>
         <property name="icon">
          <iconset resource="usr.qrc">
           <normaloff>:/res/usr/pro.png</normaloff>:/res/usr/pro.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QPushButton" name="pushButton_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(246, 209, 85,0.9);</string>
         </property>
         <property name="text">
          <string>菜单列表</string>
         </property>
         <property name="icon">
          <iconset resource="usr.qrc">
           <normaloff>:/res/usr/list.png</normaloff>:/res/usr/list.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1" rowspan="3">
        <widget class="QPushButton" name="pushButton_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 170, 255);</string>
         </property>
         <property name="text">
          <string>菜品管理</string>
         </property>
         <property name="icon">
          <iconset resource="usr.qrc">
           <normaloff>:/res/usr/adi.png</normaloff>:/res/usr/adi.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_7">
         <property name="styleSheet">
          <string notr="true">border-radius:25px;
background-color: rgb(1, 170, 128);
border:2px solid;

</string>
         </property>
         <property name="text">
          <string>服务请求</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
border-radius:10px;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <widget class="QLabel" name="label_des">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QLabel" name="label_ser">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1" rowspan="2" colspan="3">
        <widget class="QPushButton" name="pushButton_6">
         <property name="font">
          <font>
           <family>华光粗圆_CNKI</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 133, 92);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>数据库：已连接</string>
         </property>
         <property name="icon">
          <iconset resource="usr.qrc">
           <normaloff>:/res/usr/data.png</normaloff>:/res/usr/data.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="0" rowspan="2">
        <widget class="QPushButton" name="pushButton_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>华光琥珀_CNKI</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(227, 124, 255);</string>
         </property>
         <property name="text">
          <string>打印机：未连接</string>
         </property>
         <property name="icon">
          <iconset resource="usr.qrc">
           <normaloff>:/res/usr/print.png</normaloff>:/res/usr/print.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="3">
        <widget class="QPushButton" name="pushButton_5">
         <property name="font">
          <font>
           <family>华光粗圆_CNKI</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 85, 127);</string>
         </property>
         <property name="text">
          <string>网络：已连接</string>
         </property>
         <property name="icon">
          <iconset resource="usr.qrc">
           <normaloff>:/res/usr/wifi.png</normaloff>:/res/usr/wifi.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="usr.qrc"/>
 </resources>
 <connections/>
</ui>
